master/internal/core.go

```diff
@@ -488,6 +488,8 @@ func (m *Master) Run() error {
 	m.echo.Static("/swagger-ui", filepath.Join(m.config.Root, "static/swagger-ui"))
 	m.echo.Static("/api/v1/api.swagger.json",
 		filepath.Join(m.config.Root, "swagger/determined/api/v1/api.swagger.json"))
+	// Support the old experiment creation endpoint under the new API route.
+	m.echo.POST("/api/v1/experiments", api.Route(m.postExperiment), authFuncs...)
```

Comment:
What's going to happen for the swagger docs for this?

Reply:
I think we talked about this, I just forgot.

Reply:
we have https://determinedai.atlassian.net/browse/DET-4052 for that. we'd need to figure out what would be the best way of adding that. one way would be to add it through proto/scripts/swagger.py